### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by dasousarules**
I think now it should be fixed

### CODE REVIEW

File: Assignment4/dasousarules-y160423/Task06.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  print(s,p,o)
+
+
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# In[10]:
+
+
+# TO DO
+g.add((ns.JohnSmith, VCARD.Work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/dasousarules-y160423/Task06.py:95]: vcard does not define the Work property. Create it in your own namespace

File: Assignment4/dasousarules-y160423/Task07.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+for s,p,o in g.triples((None, RDFS.subClassOf, NS.Person)):
+  for ssc,psc,osc in g.triples((None, RDF.type, s)):
+    print(ssc)
+
+# sparql
+q2 = prepareQuery('''
+  SELECT ?ind WHERE { 
+    {?ind a ns:Person.}
+    UNION
+    {?sc rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/dasousarules-y160423/Task07.py:79]: use * to retrieve the full taxonomy of sublcasses

File: Assignment4/dasousarules-y160423/Task07.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  for ssc,psc,osc in g.triples((None, RDF.type, s)):
+    for i,pi,oi in g.triples((ssc, None, None)):
+      print(ssc,pi,s)
+
+# sparql
+q3 = prepareQuery('''
+  SELECT ?ind ?p ?c WHERE {
+  {
+    ?ind a ns:Person.
+    ?ind a ?c .
```
Comment (dgarijo) [inline Assignment4/dasousarules-y160423/Task07.py:115]: why ?ind a ?c here?

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by dasousarules** [comment]
Is it ok now?

**Pull request merged**